docs/_reporting/hubs/README.md

```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
 
 ## 📦 What's included
 
-The FinOps hub template includes the following resources:
+The default deployment of FinOps hub template includes the following resources:
 
 - Storage account (Data Lake Storage Gen2) to hold all cost data.
 - Data Factory instance to manage data ingestion and cleanup.
 - Key Vault to store the Data Factory system managed identity credentials.
 
-Once deployed, you can report on the data in Power BI or by connecting to the storage account directly.
+If you would like to use the new Azure Data Explorer features, you will need to enbale the parameter **dataExplorerName**. To learn more about the parameters needed to deploy Hubs with ADX, refer to the [template](../hubs/template.md) page.
+
+Once deployed, you can report on the data in ADX dashboards (__comming soon__), Power BI or by connecting to the storage account directly.
 
 <img alt="Screenshot of the cost summary report" style="max-width:200px" src="https://user-images.githubusercontent.com/399533/216882658-45f026f1-c895-48ca-81e2-35765af8e29e.png">
 <img alt="Screenshot of the services cost report" style="max-width:200px" src="https://user-images.githubusercontent.com/399533/216882700-4e04b589-0580-4e49-9b40-9f5948792975.png">
```

Comment:
I can't leave any comments below this point since nothing changed, but we should probably also update:
1. Should we explicitly call out the ADX cluster name in the Create a new hub > Deploy your FinOps hub section? That's where we want them to make sure to enable it.
2. Connect to your data should be updated to call out ADX.
3. Get started with hubs should also call out ADX.
4. The required permissions below aren't the full story, but we should confirm if the template.md file has the required permissions. I don't think I did that.